locale/zh-cn/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -323,17 +323,17 @@ myEmitter.on('event', () => {
 
 ```js
 const EventEmitter = require('events');
-const util = require('util');
 
-function MyEmitter() {
-  EventEmitter.call(this);
+class MyEmitter extends EventEmitter {
+  constructor() {
+    super();
 
-  // use nextTick to emit the event once a handler is assigned
-  process.nextTick(() => {
-    this.emit('event');
-  });
+    // use nextTick to emit the event once a handler is assigned
```

Comment:
```
// 分配处理方法后，使用 nextTick 触发事件
```

Reply:
I don't tend to translate the code comment from English to Chinese to keep it as what it is now, because we like English comments. :)